leaders.html

```diff
@@ -70,7 +70,7 @@
           <h3 class="panel-title">Leaderboard</h3>
         </div>
         <div class="panel-body">
-          <h2>To be announced.</h2>
+          <iframe src="https://sigmod-contest-25.hpi-sci.de/" width="100%" height="1200" frameborder="0"></iframe>
```

Comment:
We are constantly improving this site. If you have any comments on that part, please let us know.